src/components/propositionalLogic/wiki.tsx

```diff
@@ -20,17 +20,36 @@ class WikiPropositionalLogic extends React.Component {
 				<p>Atomare Formeln können mit den Junktoren
 				&#172; (nicht), &amp; (und), | (oder), &rarr; (Implikation) und &harr; (Äquivalenz).
 				&#172; negiert den Wahrheitswert einer Formel. Das heißt eine wahre Formel wird falsch
-				und eine falsche Formel wird wahr. &#172; ist der
-				einzige Junktor, der nur auf einer statt auf zwei Formeln
-				operiert.
+				und eine falsche Formel wird wahr. Statt &amp; sieht man normalerweise
+				das Zeichen &#8743; und statt | das Zeichen &#8744;. Auf einer gewöhnlichen Tastatur
+				gibt es diese Zeichen jedoch nicht. Daher benutzen wir in Diproche lieber die ersten
```

Comment:
Hier ist ein starker Sprachstilwechsel. Den großteil handelst Du eher fachlich ab und dann springst Du in eine lockere Alltagssprache. Wie man das bewertet ist eine andere Frage, ich habe dazu keine starke Meinung.

Auch: Nicht nur Diproche macht das so, sondern das ist eine allgemeine Handhabung.

Reply:
"Allgemeine Handhabung" würde ich so nicht unbedingt sagen. Im digitalen Kontext: Ja. Dort aber auch nicht konsequent. Es gibt z.B. Beweistools (Namen fallen mir gerade nicht ein, glaube aber REDLOG) bei denen man "und" als /\ und "oder" als V schreibt.

Was den Wechsel im Sprachstil angeht: Ja, den siehst du da richtig. Besonders fällt mir gerade auch das plötzliche kollektive "Wir" auf. Wird umformuliert.

Reply:
Stimmt. Was ich mich jetzt im Nachhinein frage ist: Wo benutzen wir diese denn? Im Texteditor und diproche selber benutzten wir die ja nicht sondern setzten "und".

Reply:
Wir selbst nicht, das stimmt. Aber es sind auch erlaubte Eingaben. Ich denke, das sollte ich auch noch aufzählen